Modules/Dashboard/Http/Controllers/ReportController.php

```
@@ -0,0 +1,583 @@
+    }
+
+    /**
+     * Get specific report details
+     */
+    public function show(int $id): JsonResponse
+    {
+        try {
+            $report = Report::with(['creator'])->findOrFail($id);
+
+            // Check access permissions
+            if (!$report->canAccess($request->user())) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$request'. (Unused Code Rules)

(UnusedLocalVariable)